### PR TITLE
Deprecate `versionInPackageJson` parameter

### PR DESCRIPTION
**Pull request by fwilhe**
Closes SAP/cloud-sdk-backlog#161.

### CODE REVIEW

File: test-packages/e2e-tests/README.md [hunk 1]
```diff
@@ -10,6 +10,6 @@ For testing if two versions works together do the following in the e2e folder:
 
 ```bash
 rm -r node_modules/ && yarn install --ignore-engines       //contains state of pm2
-rm -r sqlite.db && yarn deploy && yarn stop && yarn start  //recreate DB
+rm sqlite.db && yarn deploy && yarn stop && yarn start  //recreate DB
```
Comment (florian-richter) [inline test-packages/e2e-tests/README.md:13]:
```suggestion
rm sqlite.db && yarn deploy && yarn stop && yarn start     //recreate DB
```
[pp] Align comment with previous line

**Review by florian-richter — APPROVED** [review]
LGTM

File: test-packages/e2e-tests/test/generator-cli.spec.ts [hunk 2]
```diff
@@ -60,4 +60,23 @@ describe('generator-cli', () => {
     expect(entities).toContain('TestEntity.ts');
     expect(entities).toContain('package.json');
   }, 60000);
+
+  it('should set version when versionInPackageJson parameter is used', async () => {
```
Comment (marikaner) [inline test-packages/e2e-tests/test/generator-cli.spec.ts:64]: [req] parameter => option
```suggestion
  it('should set version when versionInPackageJson option is used', async () => {
```

**Review by marikaner — COMMENTED** [review]
Just two general remarks: 
1. We use the imperative in the changelog.
2. I think we should consistently talk about generator options, instead of arguments or parameters.

Everything else LGTM 👍

**Pull request merged**